db/storage/storage.go

```diff
@@ -45,7 +46,7 @@ func (c *Collection) Close() {
 func open(addr, dbname string) (*Storage, error) {
 	sess, err := mgo.Dial(addr)
 	if err != nil {
-		return nil, err
+		return nil, errors.New("mongodb: no reachable servers")
```

Comment:
I'd rather do ```fmt.Errorf("mongodb: %s", err)```, so we can catch other failures in mgo.Dial as well (including authentication errors).